.editorconfig

```diff
@@ -0,0 +1,23 @@
+# EditorConfig is awesome:http://EditorConfig.org
```

Comment:
This looks like a good start. I tinkered with a much more in-depth file here: https://github.com/aspnet/SignalR/commit/76174580fd79c2f06289aff177203c97ae292ed0 , but was disappointed to learn that VS doesn't enforce many of those options it seems, and neither does the compiler.